src/_posts/platform/app/task-scheduling/2000-01-01-custom-clock-processes.md

```diff
@@ -185,7 +255,30 @@ var job2 = new cron.CronJob({
 console.log('Started 2 cron jobs')
 ```
 
-With this example, the `job1` ticks every 2 minutes and the `job2` ticks every minute. Each job
-displays a message in the log of the application.
+### Enabling Custom Clock Processes
+
+Once your application has been successfully deployed, scale your custom clock
+process type to 1 to start your scheduler in its own dedicated container:
+
+```bash
+scalingo --app my-app scale clock:1
+```
+
+Moreover, since Scalingo expects a `web` process type to be defined (either by
+you in the `Procfile`, or automatically by a buildpack), there is most
+probably a `web` process type that would start with your application. **If you
+don't need it**, you can disable it by scaling the `web` process type to zero
+before or after your deployment:
+
+```bash
+scalingo --app my-app scale web:0
+```
```

Comment:
suggestion: do you want to add here a link to this page (https://doc.scalingo.com/platform/app/web-less-app)?